### DISCUSSION TITLE
presence rate limiting after just 2 hours of developement?

### DISCUSSION THREAD

**Discussion by kishansripada** [supabase/supabase#11019] [Questions]
Hi!

I just started developing with the presence feature and after around two hours of development, I was hit with status: rate limited. 

I'm not sure if this is client based or something, but it seems a little bit harsh considering I've just been developing?

What should I do to keep developing?

**Comment by GaryAustin1** [comment]
Have you looked thru this to see anything you might be hitting?   Also about looking in the realtime logs?
https://supabase.com/docs/guides/realtime/rate-limits

**Reply by kishansripada** [reply]
I've made sure that I included an "events per second" parameter and I limited it to one event per second. I also made sure that the realtime client is not called and created many times unnecessarily.

I'm using next-auth helpers and still have no idea why I'm being rate limited. This only happens when using presence. All other realtime features work just fine and I'm using presence exactly how the code example is. 

Are there any other suggestions for why I'm being rate limited?

**Reply by kishansripada** [reply]
Also When i check the realtime logs, they are completely empty
